pkg/appconsts/consensus_consts.go

```diff
@@ -5,4 +5,5 @@ import "time"
 const (
 	TimeoutPropose = time.Second * 10
 	TimeoutCommit  = time.Second * 11
+	GoalBlockTime  = time.Second * 15
```

Comment:
[extremely optional]

```suggestion
	// GoalBlockTime is the target time interval between blocks. Since the block
	// interval isn't enforced at consensus, the real block interval isn't
	// guaranteed to exactly match GoalBlockTime. GoalBlockTime is currently targeted
	// through static timeouts (i.e. TimeoutPropose, TimeoutCommit).
	GoalBlockTime = time.Second * 15
```